src/v3/test/integration/authenticator-expired-password-no-complexity.test.tsx

```diff
@@ -25,15 +25,15 @@ describe('authenticator-expired-password-no-complexity', () => {
 
   it('should send correct payload', async () => {
     const {
-      authClient, user, findByTestId, findByText,
+      authClient, user, findByText, findByLabelText,
     } = await setup({ mockResponse });
 
     const titleElement = await findByText(/Your password has expired/);
     await waitFor(() => expect(titleElement).toHaveFocus());
 
     const submitButton = await findByText('Change Password', { selector: 'button' });
-    const newPasswordEle = await findByTestId('credentials.passcode') as HTMLInputElement;
```

Comment:
Why can't we use `findByTestId ` ?

Reply:
Same answer as: https://github.com/okta/okta-signin-widget/pull/3527#discussion_r1476660419